releasetool/commands/start/go.py

```diff
@@ -0,0 +1,206 @@
+def determine_last_release(ctx: Context) -> None:
+    click.secho("> Figuring out what the last release was.", fg="cyan")
+    if ctx.relative_module_name is None:
+        prefix = "v"
+    else:
+        prefix = ctx.relative_module_name + "/v"
+    tags = releasetool.git.list_tags()
+    candidates = [
+        tag
+        for tag in tags
+        if tag.startswith(prefix)
+    ]
+
+    if candidates:
+        ctx.last_release_committish = candidates[0]
+        ctx.last_release_version = candidates[0].rsplit("/").pop()[1:]
+
+    else:
+        click.secho(
+            f"I couldn't figure out the last release for {ctx.module_name}, "
+            "so I'm assuming this is the first release. Can you tell me "
+            "which git rev/sha to start the changelog at?",
+            fg="yellow",
+        )
+        ctx.last_release_committish = click.prompt("Committish")
+        ctx.last_release_version = "0.0.0"
+
+    click.secho(f"The last release was {ctx.last_release_version}.")
+
+
+def gather_changes(ctx: Context) -> None:
+    click.secho(f"> Gathering changes since {ctx.last_release_version}", fg="cyan")
+    ctx.changes = releasetool.git.summary_log(
+        from_=ctx.last_release_committish, to=f"origin/master"
+    )
+    click.secho(f"Cool, {len(ctx.changes)} changes found.")
+
+
+def determine_release_version(ctx: Context) -> None:
+    click.secho(f"> Now it's time to pick a release version!", fg="cyan")
+    release_notes = textwrap.indent(ctx.release_notes, "\t")
+    click.secho(f"Here's the release notes you wrote:\n\n{release_notes}\n")
+
+    parsed_version = [int(x) for x in ctx.last_release_version.split(".")]
+
+    if parsed_version == [0, 0, 0]:
+        ctx.release_version = "0.1.0"
+        return
+
+    selection = click.prompt(
+        "Is this a major, minor, or patch update (or enter the new version " "directly)"
+    )
+    if selection == "major":
+        parsed_version[0] += 1
+        parsed_version[1] = 0
+        parsed_version[2] = 0
+    elif selection == "minor":
+        parsed_version[1] += 1
+        parsed_version[2] = 0
+    elif selection == "patch":
+        parsed_version[2] += 1
+    else:
+        ctx.release_version = selection
+        return
+
+    ctx.release_version = "{}.{}.{}".format(*parsed_version)
+
+    click.secho(f"Got it, releasing {ctx.release_version}.")
+
+
+def create_release_branch(ctx) -> None:
+    if ctx.module_name is None:
+        ctx.release_branch = f"release-{ctx.release_version}"
+    else:
+        ctx.release_branch = f"release-{ctx.module_name}-{ctx.release_version}"
+    click.secho(f"> Creating branch {ctx.release_branch}", fg="cyan")
+    return releasetool.git.checkout_create_branch(ctx.release_branch)
+
+
+def update_changelog(ctx: Context) -> None:
+    click.secho(f"> Updating {_CHANGELOG_FILENAME}.", fg="cyan")
+
+    if not os.path.exists(_CHANGELOG_FILENAME):
+        print(f"{_CHANGELOG_FILENAME} does not yet exist. Opening it for " "creation.")
```

Comment:
Should this print be click.secho?

Reply:
Copied from the python start command. Will let @theacodes answer.

Reply:
Either or is fine. Use secho generally for steps. Helpful messages during those steps can just be a print.